app/src/main-process/ordered-webrequest.ts

```diff
@@ -140,7 +140,7 @@ export class OrderedWebRequest {
     this.onBeforeRequest = new AsyncListenerSet(
       webRequest.onBeforeRequest.bind(webRequest),
       async (listeners, details) => {
-        let response: Response = {}
+        let response: CallbackResponse = {}
```

Comment:
Same update to api as explained in https://github.com/desktop/desktop/pull/15828